modules/rules_android_ndk/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "homepage": "https://github.com/bazelbuild/rules_android_ndk",
+  "maintainers": [
+    {
+      "email": "bcr-maintainers@bazel.build",
+      "name": "No Maintainer Specified"
```

Comment:
Could you add yourself as a maintainer here?

Reply:
Done